### PR TITLE
DB Setup and API Implementations

### PR DESCRIPTION
**Pull request by yuting0203**
## What

Added .gitignore for environment setup 
Created SQLite to store item information and modified POST /items and GET /items endpoints for accessing the database
Added GET /search endpoint to search for items

## CHECKS :warning:

Please make sure you are aware of the following.

- [ ] **The changes in this PR doesn't have private information

### CODE REVIEW

File: python/main.py [hunk 1]
```diff
@@ -19,34 +23,44 @@
...
 
-    with open('items.json', 'r+') as jsonfile:
-        items = json.load(jsonfile)
-        items['items'].append({
-            'name': name,
-            'category': category
-        })
+    conn = sqlite3.connect(DBPATH)
+    c = conn.cursor()
+    c.execute("INSERT INTO items VALUES (NULL,?,?)", item)
```
Comment (tsoibet) [inline python/main.py:47]: You may choose which columns to insert data by adding parentheses with column names in it:
`INSERT INTO items(column_name_1, column_name_2, ... etc) VALUES (?, ?, ...)`
This way you do not have to use "NULL" in VALUES.

File: python/main.py [hunk 2]
```diff
@@ -62,6 +62,17 @@ def get_item():
 
     return {"item":items}
 
+@app.get("/search")
+def get_item(keyword):
+    logger.info(f"Search {keyword} items")
+    conn = sqlite3.connect(DBPATH)
+    conn.row_factory = dict_factory
+    c = conn.cursor()
+    c.execute("SELECT * FROM items WHERE name=?;", [keyword])
```
Comment (tsoibet) [inline python/main.py]: Let's assume there's an item "long skirt". If you search "skirt", will the item "long skirt" be returned?

https://www.sqlitetutorial.net/sqlite-like/
The "LIKE" operator may be useful in this case.

File: python/main.py (hunk 1 above)
Comment (yuting0203) [reply python/main.py:47]: Thank you for the advice.

File: python/main.py (hunk 2 above)
Comment (yuting0203) [reply python/main.py]: Thank you for pointing out this!!!
I did not notice the requirement is "including" the keyword.

**Review by tsoibet — APPROVED** [review]
LGTM

**Pull request merged**